common/settings.h

```diff
@@ -37,6 +37,7 @@ struct Settings {
   std::vector<std::string> dart_flags;
   std::string log_tag = "flutter";
 
+  static bool using_blink;
```

Comment:
This value should not be a static.  Instead, it should be a member variable, like all the other settings.